flexget/plugins/api_trakt.py

```diff
@@ -851,15 +847,24 @@ def lookup_movie(session=None, only_cached=None, **lookup_params):
     @staticmethod
     def collected(style, trakt_data, title, username=None, account=None):
         ident = str(account) + '|' + str(username)
+        style_ident = 'movies' if style == 'movie' else 'shows'
         add_user_to_cache(ident)
-        if not ApiTrakt.user_cache[ident]['collection'][style + 's']:
-            update_collection_cache(style, ident, username=username, account=account)
-        if not ApiTrakt.user_cache[ident]['collection'][style + 's']:
+        if not ApiTrakt.user_cache[ident]['collection'][style_ident]:
+            update_collection_cache(ident, style_ident, username=username, account=account)
+        if not ApiTrakt.user_cache[ident]['collection'][style_ident]:
             log.warning('No collection data returned from trakt.')
             return
         in_collection = False
-        cache = ApiTrakt.user_cache[ident]['collection'][style + 's']
+        cache = ApiTrakt.user_cache[ident]['collection'][style_ident]
         if style == 'show':
+            if trakt_data.id in cache:
+                log.error('work')
+                series = cache[trakt_data.id]
+                num = 0
+                for s in series['seasons']:
+                    num += len(s['episodes'])
+                in_collection = num == trakt_data.aired_episodes
```

Comment:
Do either of these numbers take into account specials? What should the behavior with specials be? Should you need every special for it to be marked collected? I think trakt's builtin behavior says they don't matter.